argus/cbinit/base_config_generator.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2016 Cloudbase Solutions Srl
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import abc
+import ConfigParser
+import ntpath
+import StringIO
+import tempfile
+
+import six
+
+from argus import util
+
+
+def _get_base_conf(config_name):
+    """Return a ConfigParser object withe default values."""
+    base_conf = StringIO.StringIO(
+        util.get_resource(config_name))
```

Comment:
The error should stop the test, this base configuration file must be present in the resources folder, if they are not present then there is a logical problem and we cannot continue the test. The best I can do is to wrap the `IOError` that we can get in a` ArgusError` but I don't see the point because we cannot go around this problem if the file is not present.